tests/hpc/mpi_master.pm

```diff
@@ -25,65 +25,66 @@ sub run ($self) {
     select_serial_terminal();
     my $mpi = $self->get_mpi();
     my ($mpi_compiler, $mpi_c) = $self->get_mpi_src();
-    my $mpi_bin = 'mpi_bin';
-    my $mpi2load = '';
+    my $mpi_bin = "mpi_bin";
```

Comment:
personally i am against this change. I think we should not use double quotes whenever it is not required. I usually dont complain when someone make some change and do so but IMO this is unnecessary here. it makes also easier to see whether some string does interpolation or not. 
I would like to ask you to exclude this commit. is that ok?

Reply:
It's kinda personal style, so let's look on programming style 
 at https://en.wikibooks.org/wiki/Perl_Programming/Strings#Single-quoted_strings


Putting the value of $name into the string in the first print statement is called "interpolation." **If you don't need interpolation, you should use single quotes, because it makes your intent clearer.**

Reply:
ok